README.rst

```diff
@@ -28,6 +28,10 @@ Please open a GitHub issue for any feedback or issues:
 https://github.com/awslabs/cfncluster.  There is also an active AWS
 HPC forum which may be helpful:https://forums.aws.amazon.com/forum.jspa?forumID=192.
 
+CfnCluster 1.5 IAM Change
+=========================
+Between CfnCluster 1.4.2 and 1.5.0 we made a change to the CfnClusterInstancePolicy that adds “ec2:DescribeVolumes” permissions. If you’re using a custom policy (e.g. you specify "ec2_iam_role" in your config) you’ll need to make this change in your policy as well.
```

Comment:
I'd slightly reword it  to say that is not necessarily needed to change the policy if it already includes the required permissions, something like    "If you’re using a custom policy (...) be sure it already includes the new required permission or you’ll need to make this change in your policy as well"

Reply:
done